tests/includeme_test.py

```diff
@@ -23,8 +23,19 @@ def test_bad_schema_validated_on_include():
         'pyramid_swagger.schema_directory': 'tests/sample_schemas/bad_app/',
     }
     mock_config = mock.Mock(registry=mock.Mock(settings=settings))
-    with pytest.raises(jsonschema.exceptions.ValidationError):
+    with pytest.raises(ValidationError) as excinfo:
         pyramid_swagger.includeme(mock_config)
+    assert "'nickname' is a required property" in str(excinfo.value)
+
+
+@mock.patch('pyramid_swagger.register_api_doc_endpoints')
+@mock.patch('pyramid_swagger.get_swagger_schema')
+def test_2dot0_schema_validated_on_include(_, _mock):
+    settings = {
+        'pyramid_swagger.schema_directory': 'tests/sample_schemas/2.0/',
+    }
+    mock_config = mock.Mock(registry=mock.Mock(settings=settings))
```

Comment:
Please include `spec=<type>` as an argument so it is easy to infer the expected type. This also has the benefit of making sure you don't deviate from the type's expected interface.